peachjam/forms.py

```diff
@@ -197,3 +201,44 @@ class AttachedFilesForm(AttachmentFormMixin, forms.ModelForm):
     class Meta:
         model = AttachedFiles
         fields = "__all__"
+
+
+class DocumentProblemForm(forms.Form):
+    document_link = forms.CharField(max_length=255, required=True)
+    problem_description = forms.CharField(widget=forms.Textarea, required=True)
+    email_address = forms.EmailField(required=False)
+
+    def send_email(self):
```

Comment:
Nice -- putting it here moves the logic that works with the data, close to the data.